ddtrace/internal/remoteconfig/__init__.py

```diff
@@ -13,14 +16,18 @@ class RemoteConfig(object):
 
     @classmethod
     def enable(cls):
-        # type: () -> None
-        with cls._worker_lock:
-            if cls._worker is None:
-                cls._worker = RemoteConfigWorker()
-                cls._worker.start()
+        # type: () -> bool
+        rc_env_enabled = asbool(os.environ.get("DD_REMOTE_CONFIGURATION_ENABLED", "true"))
```

Comment:
This will require both `DD_REMOTE_CONFIGURATION_ENABLED` and `DD_DYNAMIC_INSTRUMENTATION_ENABLED` to be set to true for DI to work, which is not what we support and document 🙁

Reply:
IIUC, if ``DD_REMOTE_CONFIGURATION_ENABLED`` wasn't set, it would treat it as True. This is to avoid it from starting if it is explicitly set to False.

Reply:
``DD_REMOTE_CONFIGURATION_ENABLED`` is True by default since https://github.com/DataDog/dd-trace-py/pull/4813